### PR TITLE
Added describe_Provisioned_product.sl file + sl tests.

### PR DESCRIPTION
**Pull request by LenutaToderean**
Signed-off-by: lenuta-toderean <lenuta.toderean@hpe.com>

Added describe_Provisioned_product.sl file + sl tests.

### CODE REVIEW

File: content/io/cloudslang/amazon/aws/servicecatalog/products/describe_provisioned_product.sl [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+#!                                                             valid results. Wait for an AVAILABLE status before
+#!                                                             performing operations.
+#!                                            'TAINTED' - Stable state, ready to perform any operation. The stack has
+#!                                                        completed the requested operation but is not exactly what was
+#!                                                        requested. For example, a request to update to a new version
+#!                                                        failed and the stack rolled back to the current version.
+#!                                            'ERROR' - An unexpected error occurred, the provisioned product exists but
+#!                                                      the stack is not running. For example, CloudFormation received a
+#!                                                      parameter value that was not valid and could not launch the stack.
+#!                                            'PLAN_IN_PROGRESS'
```
Comment (mariuspetrovan) [inline content/io/cloudslang/amazon/aws/servicecatalog/products/describe_provisioned_product.sl]: no description for 'plan in progres' ?

File: content/io/cloudslang/amazon/aws/servicecatalog/products/describe_provisioned_product.sl [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+#!                Optional
+#! @input accepted_language: The language code.
+#!                           Example: en (English), jp (Japanese), zh(Chinese)
+#!                           Default: 'en'
+#!                           Optional
+#! @input product_id: The provisioned product identifier.
+#!                    Required
+#!
+#! @output return_code: "0" if operation was successfully executed, "-1" otherwise.
+#! @output return_result: The full API response in case of success, or an error message in case of failure.
```
[the remark this answers is not in the document]
Comment (LenutaToderean) [reply content/io/cloudslang/amazon/aws/servicecatalog/products/describe_provisioned_product.sl]: Yes. It is in JSON format.

File: content/io/cloudslang/amazon/aws/servicecatalog/products/describe_provisioned_product.sl (hunk 1 above)
Comment (LenutaToderean) [reply content/io/cloudslang/amazon/aws/servicecatalog/products/describe_provisioned_product.sl]: No desc for this state in the official doc.

Comment (mariuspetrovan) [reply content/io/cloudslang/amazon/aws/servicecatalog/products/describe_provisioned_product.sl]: other than that, looks good 👍

**Pull request merged**